README.md

```diff
@@ -84,12 +87,14 @@ jobs:
 
       - name: Process trx reports with default
         if: always()
-        uses: im-open/process-dotnet-test-results@v2.2.6
+        # You may also reference just the major or major.minor version
+        uses: im-open/process-dotnet-test-results@v2.2.7
```

Comment:
Even though its default doesn't change behavior, I think this should be a minor version release because it changes the API.

Reply:
Ok I will commit with minor.

Reply:
Ok committed with minor version number.